lib/Db/TagMapper.php

```diff
@@ -248,4 +225,26 @@ public function createDefaultTags(MailAccount $account): void {
 			$this->insert($entity);
 		}
 	}
+
+	public function deleteOrphans() {
+		$qb = $this->db->getQueryBuilder();
+		$subQb1 = $this->db->getQueryBuilder();
+		$dqb = $this->db->getQueryBuilder();
+
+		// get lowest ids
+		$groupByQuery = $subQb1->select($qb->func()->min('id'))
+			->from('mail_message_tags')
+			->groupBy('imap_message_id','tag_id')
+			->having('COUNT(id)>1');
```

Comment:
use ``\OCP\DB\QueryBuilder\IQueryBuilder::func`` and ``\OCP\DB\QueryBuilder\IFunctionBuilder::count`` to generate the COUNT(id) and the expression builder for the >1